dev-db/sqlitestudio/sqlitestudio-3.4.3.ebuild

```
@@ -132,6 +164,12 @@ src_compile() {
 }
 
 src_install() {
+	if use test; then
+		# remove test artifacts that must not be installed
+		rm -rf "${D}"/lib64
+		rm -rf "${D}"/usr/share/qt5/tests
```

```suggestion
		rm -r "${ED}"/usr/share/qt5/tests || die
```
We tend to not use `rm -f` and add `|| die` to check the result.  This way, if in a future version this becomes unnecessary, we will know and remove it.

Also using `ED` instead of `D` helps for prefix installs.

---

Had a look at a random sample of ebuilds and found no clear pattern. Some do exactly this. Perfectly fine for me to adjust this. I'd appreciate a styleguide with standard patterns. Lowers discussion- and review-effort. (Went only through some of the Wiki- and blog-sources.)